package.json

```diff
@@ -28,7 +28,9 @@
 	],
 	"scripts": {
 		"build": "tsup src/index.ts --clean --format cjs,esm --outDir lib --dts && cp lib/index.d.ts lib/index.d.cts",
-		"docs": "typedoc",
+		"docs": "pnpm run docs:build && pnpm run docs:build-post",
+		"docs:build": "typedoc",
+		"docs:build-post": "find -lname \"$PWD/docs/generated/*\" | while read link; do target=$(readlink $link); relative_target=$(realpath $target --relative-to=$link/.. --no-symlinks --canonicalize-missing); ln --force --symbolic --no-target-directory $relative_target $link; done",
```

Comment:
Uhh, this is a lot of shell scripting and magics. What's going on here?

Reply:
All it does is convert the absolute symlinks to relative symlinks. I open an issue here to see if we can get this done nicer: https://github.com/citkane/typedoc-plugin-versions/issues/29